tests/local_mode/test_language.py

```diff
@@ -38,8 +38,12 @@ def test_vector_valid_token_is_not_zero():
     """Test that the vector of a valid token is not all zeros"""
 
     doc = nlp("banana")
-    actual = doc[0].vector
-    zeros = np.zeros(actual.shape)
+    # actual = doc[0].vector
+    # Convert numpy vector from doc var to torch tensor
+    actual = torch.tensor(doc[0].vector, dtype=torch.float32)
```

Comment:
This conversion should not be required, the vector is expected to be torch.Tensor